logstash_transit/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM docker.elastic.co/logstash/logstash:7.2.0
+
+LABEL maintainer "Robert Kenny <R.Kenny@wellcome.ac.uk>"
+LABEL description "A Docker image for deploying logstash"
+
+USER 0
+
+# Installs envsubst on Centos
+RUN yum install -y gettext
+
+USER 1000
```

Comment:
What's this?

Reply:
That's the original user ID the parent dockerfile sets - TLDR: I don't know.